src/api/models/mermaid.py

```diff
@@ -683,6 +684,19 @@ class Meta:
         db_table = 'quadrat_collection'
 
 
+class TransectQuadrat(BenthicTransect):
+    project_lookup = "sample_event__site__project"
+
+    quadrat_size = models.DecimalField(
+        decimal_places=2, max_digits=6,
+        verbose_name=_(u'single quadrat area (m2)'),
```

Comment:
search/replace unicode

Reply:
I only cleaned up the ones in classes that I directly touched because there were over 100 instances when I did a search for `u'`.  I'll add it to the black/flake8/isort card as a clean-up item.